### DISCUSSION TITLE
[Feature] Allow comments/notes on Contact Points

### DISCUSSION THREAD

**Discussion by laundmo** [grafana/grafana#67524] [Alerting]
We have multiple contact points which use email, but to send SMS.
For this we use emails in the format `0123456789@provider.com`
It's very hard to distinguish these when we have many sms-emails in the same contact point. Also, we need to consult a separate table of phone numbers to map these to the full name or role of the person, as the number itself is meaningless when editing the contact point.

I would like to be able to store this information on the specific Contact Point.

This is not specific to Email, as a Webhook Integration URLs are often similarily meaningless to know what the destination is. i'm sure any Integration has this issue in some form.

Solution Ideas:
- Add a comment/note field to all Contact Point Integrations by default.
- Instead of `;`separated, allow adding emails in a table. Then a second column can contain metadata about this Email (specific to Email integration, not ideal for other Integrations)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82748. Feel free to continue the discussion around this there. Thank you!
